index.html

```diff
@@ -820,8 +820,13 @@ <h3>
           <li>[=list/For each=] |rawInputIndex:long| of [=the range=] from 0 to
           |inputCount| − 1:
             <ol>
-              <li>If the the gamepad button at index |rawInputIndex|
-              [=represents a Standard Gamepad button=]:
+              <li>Let |button| be the button at index |rawInputIndex|.
+              </li>
+              <li>Let |type| be the result of determining if the |button| [=represents
+              a Standard Gamepad button=] or represents [=Additional gamepad buttons=]
```

Comment:
Let's use singular in both cases and add a period at the end of the sentence:

```
a Standard Gamepad button=] or represents an [=additional gamepad button=].
```

I'm not sure if respec will link the definition correctly if we don't use "Additional gamepad buttons"; if not, we can add a data-lt attribute to the <dfn> tag below:

```
<dfn data-lt="additional gamepad button">Additional gamepad buttons</dfn>
```